src/main/scala/inox/solvers/smtlib/SMTLIBParser.scala

```diff
@@ -106,6 +106,9 @@ trait SMTLIBParser {
     case SString(value) =>
       StringLiteral(utils.StringUtils.decode(value))
 
+    case FunctionApplication(QualifiedIdentifier(SimpleIdentifier(SSymbol("seq.unit")), None), Seq(SHexadecimal(hex))) =>
+      StringLiteral(utils.StringUtils.decode(hex.repr))
+
```

Comment:
Isn't this a z3-specific representation? IIRC CVC4 uses a native representation for strings but Z3 uses `seq` internally. I'm not really sure anymore though.

Reply:
Yes I think you're right, we can always move it back here later if needed.